tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -286,7 +352,22 @@
     return (node.metrics && node.metrics.time)
               ? tf_op_profile.percent(node.metrics.time) : 0;
   },
-  _selectedChanged: function(v) { this.classList.toggle('selected', v); }
+  _selectedChanged: function(v) { this.classList.toggle('selected', v); },
+  /*
+    Returns top K children for the node. If p90 is true, then only children
+    fallen in the top 90th percentile in time are returned. 
+  */
+  _getKChildren: function(node, k, p90) {
+    if(p90 && node.children.length > 0 && node.children[0].metrics){
+      var tot = 0, i = 0;
+      for (; i < Math.min(k, node.children.length); i++) {
+          if(tot >= node.metrics.time * 0.9) break;
+          tot += node.children[i].metrics.time;
+      }
+      k = i;
+    }
+    return this.level ? node.children.slice(0, k) : node.children;
```

Comment:
Consider making this method take `level` as a parameter, ie `function(node, k, p90, level)` instead of using `this.level` within the function. That way, `level` is guaranteed to be set, and `level` would only be false here if it equalled 0.

Reply:
Done.